terraform/cloudfoundry/dns-records.tf

```diff
@@ -82,6 +82,14 @@ resource "aws_route53_record" "logsearch" {
   records = ["${aws_elb.logsearch_kibana.dns_name}"]
 }
 
+resource "aws_route53_record" "logsearch_ingestor" {
+  zone_id = "${var.system_dns_zone_id}"
+  name    = "logsearch-ingestor.${var.system_dns_zone_name}."
+  type    = "CNAME"
```

Comment:
AWS recommends to use alias type for ELBs...
http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-choosing-alias-non-alias.html